tests/client/core/TestInstallAddon.js

```diff
@@ -96,6 +97,10 @@ describe('withInstallHelpers inner functions', () => {
     mapDispatchToProps = makeMapDispatchToProps({ WrappedComponent, src });
   });
 
+  beforeEach(() => {
+    configStub = sinon.stub(config, 'get').withArgs('server').returns(false);
```

Comment:
Or we can turn on resetModules to always be run, then you can mock modules without cause for concern. See https://facebook.github.io/jest/docs/en/configuration.html#resetmodules-boolean

Also I've not personally done this until now, but I notice it is already used elsewhere in the test suite.